searchlib/src/vespa/searchlib/attribute/multi_value_mapping_base.h

```diff
@@ -39,6 +39,8 @@ class MultiValueMappingBase
     void updateValueCount(size_t oldValues, size_t newValues) {
         _totalValues += newValues - oldValues;
     }
+
+    EntryRef acquire_multi_value_entry_ref(uint32_t docId) const noexcept { return _indices.acquire_elem_ref(docId).load_acquire(); }
```

Comment:
As this is inside MultiValueMappingBase, perhaps we should shorten the function name to _acquire_entry_ref()_ ?